package.json

```diff
@@ -36,6 +37,7 @@
     "graphql": "^0.6.0",
     "lodash": "^4.13.1",
     "newrelic": "^1.28.3",
+    "node-fetch": "^1.6.0",
```

Comment:
We do need this, but it needs to be in a in the sub directory rather than the parent so it gets packaged into the lambda:)